src/applications/registry.json

```diff
@@ -834,5 +834,16 @@
       "vagovprod": false,
       "layout": "page-react.html"
     }
+  },
+  {
+    "appName": "Healthcare Questionnaire",
+    "entryName": "questionnaire",
+    "rootUrl": "/healthcare/questionnaire",
+    "template": {
+      "vagovprod": true,
```

Comment:
Do we want this on prod? We have a flag for every environment if that is helpful.

Reply:
Yes, its behind a feature flag. and it needs to be `in production` then e2e pass

Reply:
Usually if we have something in production that users aren't using we would skip the e2e test that is what we do for `caregivers` but if its behind flipper its fine.